tasks.py

```diff
@@ -821,3 +821,26 @@ def checkchanges(ctx, action="check uncommitted files"):
         print("{} must be committed".format(res))
         raise Exit(message="#### Uncommitted files found, you may need to {} ####\n".format(action))
 
+@task(help={
+          "tag": "Docker image tag to use. Default 'cr-generator'.",
+      })
+def build_cr_generator(ctx, tag="cr-generator"):
```

Comment:
does this add something on top of the other `build` task?

Reply:
it is different from the other build task by not creating the image for every arch.